inst/tutorials/07-probability1/07-probability1.Rmd

```diff
@@ -102,7 +109,8 @@ p_pos_nd <- 0.2 ## P(Pos | no disease)
 question_text(
   "Probability (to the nearest 0.01)",
   answer("0.05", correct = TRUE),
-  answer(".05", correct = TRUE)
+  answer(".05", correct = TRUE),
+  allow_retry = TRUE
```

Comment:
@MafaldaPratas This makes it possible for students to retry on a different answer.